samples/kotlin-android-app/src/main/java/com/amplitude/android/sample/MainActivity.kt

```diff
@@ -20,7 +20,7 @@ class MainActivity : AppCompatActivity() {
             .set("custom-properties", "sample")
         val options = EventOptions()
         options.plan = Plan(branch = "test")
-        amplitude.identify(identify, options)
+        amplitude.identify(identify)
```

Comment:
Did you try if the event gets merged in identify call when we pass options in identify call? 
`amplitude.identify(identify, options)`
And does it show in the event payload?